dev-util/bpftool/bpftool-9999.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LLVM_COMPAT=( {15..19} )
+LLVM_OPTIONAL=1
```

Comment:
>       LLVM_REQUIRED_USE (GENERATED BY ECLASS)
>              An eclass-generated REQUIRED_USE string that enforces selecting exactly one slot.  It >LLVM_OPTIONAL is set, it needs to be copied into REQUIRED_USE, under appropriate  USE
>              conditions.  Otherwise, it is added automatically.

so we need (down below) `REQUIRED_USE="llvm? ( ${LLVM_REQUIRED_USE} )"`

Reply:
Good catch, added to both 9999 and 7.4.0. Thanks.